server.js

```diff
@@ -0,0 +1,17 @@
+const express = require('express');
+const app = express();
+const port = process.env.PUBLIC_PORT || 3000;
+
+// define the ping route with the response in JSON
+app.get('/ping', (req, res) => {
```

Comment:
Consider handling errors in your API endpoint. This can help improve the reliability of your application. [important]